### DISCUSSION TITLE
Code style

### DISCUSSION THREAD

**Discussion by bocops** [andregasser/bigbone#30] [General]
I've noticed that code style isn't completely consistent in what we have so far, especially regarding indentation (tab vs. spaces, and how many of them), but probably also brace placement and related things.

I prefer [OTBS](https://en.wikipedia.org/wiki/Indentation_style#Variant:_1TBS_(OTBS)) with 4-space indentation, but wouldn't mind working with anything else if there are other preferences. We probably just should decide one way or the other early on, so that we don't re-indent each others code too often. ;) Any thoughts?

**Comment by andregasser** [comment]
Hello @bocops That works for me. I'd also favor 4-space indentation over tabs and braces set as shown in your example. I'll check detekt configuration and enable checkstyle for Java code so that it is executed on pull request creation and merges into master. As well we could then trigger it by running a gradle check command locally. Does that sound ok?

I've added https://github.com/andregasser/mastodon4j/issues/32 to the list of issues.

**Reply by bocops** [reply]
Sounds good!

**Reply by andregasser** [reply]
Also added this: https://github.com/andregasser/mastodon4j/issues/34 Works complementarily to checkstyle.
